processes.go

```diff
@@ -98,182 +56,83 @@ func (c Command) String() string {
 	return strings.Join([]string(c), " ")
 }
 
-// Process holds configuration information about a Process Type.
+// Process holds configuration information about a Process.
 type Process struct {
-	ReleaseID string
-	ID        string
-	Type      ProcessType
-	Quantity  int
-	Command   Command
-	Constraints
+	Command  Command              `json:"Command,omitempty"`
```

Comment:
The `Process` type is just part of the `Formation` type, which is a `map[string]Process`. The `Formation` gets json encoded and stored in the `formation` column in postgres.